components/molecules/InsightRow/insight-row.tsx

```diff
@@ -99,18 +100,18 @@ const InsightRow = ({ insight, user, isEditable = true }: InsightRowProps) => {
               {canEdit ||
                 (!insight.is_featured && (
                   <Link href={`/hub/insights/${insight.id}/edit`}>
-                    <span className=" bg-light-slate-1 inline-block rounded-lg p-2.5 border mr-2 cursor-pointer">
+                    <span className="bg-light-slate-1 inline-block rounded-lg p-2.5 border mr-2 cursor-pointer">
                       <BsPencilFill title="Edit Insight Page" className="text-light-slate-10 text-lg" />
                     </span>
                   </Link>
                 ))}
-              <button
-                // onClick={() => handleOnDeleteClick(insight.name, insight.id)}
-                className="inline-block p-3 mr-2 border rounded-lg cursor-pointer bg-light-slate-1"
-                type="button"
-              >
-                <RiDeleteBinLine title="Delete Insight" className="text-lg text-light-slate-10" />
-              </button>
+              {handleOnDeleteClick && (
+                <div onClick={() => handleOnDeleteClick(insight.name, insight.id)}>
```

Comment:
thank you for your asking.
I have changed from div to button.
That's because if I will use button, delete button's width is wider(+2 px) than other button.
I think it is default button style.

Reply:
Hum, tried the button and it looks good on my end. Also, it's better to use a button because it is semantically correct and accessible.

<img width="825" alt="SCR-20240205-nvbn" src="https://github.com/open-sauced/app/assets/79809121/45250407-48f3-4878-af5f-1908726952b3">

Give it a second shot by adding `items-center gap-2` to the parent div and removing the `mr-2`. Do let me know if you're not getting the expected result